cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/BreakTransformer.java

```diff
@@ -41,6 +42,11 @@
     }
 
     @Override
+    public TransformModellingResult<List<String>> transform(List<String> rawData, SensitivityLevel sensitivityLevel) {
```

Comment:
IMHO you should not return null here
`1`: in the transformers that are not relevant to the sensitivity level, return the `transform(rawData)` and call the transform uniformly in TransformersHandler
`2`: leave transformers handler as-is right now, and throw NotImplementedException, NotSupportedException here

@lucian-cm WDYT?